src/unlimited.cpp

```diff
@@ -1336,7 +1336,7 @@ UniValue getstat(const UniValue& params, bool fHelp)
     if (params.size() < (verbose ? 4 : 3)) count = 1;  // if a count is not specified, give the latest sample
     else
       {
-	if (!params[3].isNum()) 
+	if (!params[(verbose ? 3 : 2)].isNum()) 
 	  {
```

Comment:
The original type for 'verbose' was an integer not bool. But I get your point. Maybe something like:
inline verboseFn(int x) { return (verbose ? (x+1) : x) ;} 
.
 params[verboseFn(2)];